zulip/zulip/cli.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+import logging
+import sys
+
+from typing import Any, Dict, List
+
+import zulip
+import click
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+log = logging.getLogger("zulip-cli")
+
+client = zulip.Client(config_file="~/zuliprc")
```

Comment:
This seems like a poor location for a config file, especially since it can’t be overridden.

Reply:
Not sure the best way to handle this. There is already https://github.com/zulip/python-zulip-api/blob/717a549f4b097b7d08470bb860f0346c30d4ecd8/zulip_bots/zulip_bots/run.py#L55, but it's buried down inside zulip_bots. Should we move it to zulip library instead?

Reply:
I think we should just add the argparse logic from the core library for how to find a zuliprc file.